src/app/component/user/user.module.ts

```diff
@@ -49,7 +49,6 @@ import {
   EditProfileComponent,
   PersonalPhotoComponent,
   SocialNetworksComponent,
-  ProfilePrivacyComponent
 } from './components';
 import { CommonModule } from '@angular/common';
```

Comment:
-- `angular` import between components.
So, organize imports:
order `angular` imports first, next other third party libs and only then imports of your application.

Reply:
fixed